addons/product/migrations/8.0.1.1/post-migration.py

```diff
@@ -78,6 +78,24 @@ def create_properties(cr, pool):
         AND name = 'standard_price'""")
     openupgrade.logged_query(cr, sql)
 
+    # Affect history to the template's company
+    # Note, for the time being, in a multi company context the history will
+    # be incomplete for products  that don't belong to a given company
+    # (Global products), because the history will be associated to the
+    # company of the SUPERUSER, during the migration
+    cr.execute("""
+        UPDATE product_price_history pph
```

Comment:
This can be done without subquery:
```
UPDATE product_price_history pph
SET company_id = product_template.company_id
FROM product_template
WHERE pph.product_template_id = product_template.template_id
AND product_template.company_id IS NOT NULL
```

Reply:
Hi @pedrobaeza, 

thanks for your review.

> Can you point in the analysis work which field change motivates this extra query?

this is a new table, populated automaticaly by ORM, when creating properties (here : https://github.com/OCA/OpenUpgrade/blob/8.0/addons/product/migrations/8.0.1.1/post-migration.py#L71)


For that reason, in the xxx_work file, there is "nothing to do" (https://github.com/OCA/OpenUpgrade/blob/8.0/addons/product/migrations/8.0.1.1/openupgrade_analysis_WORK.txt#L36), but there are some things to fix ;-)

> I also think that the method create_properties is not the best place for adding this.

This is the right place to write this extra SQL query. There is another query to fix datetime, just after the code I added. ref : https://github.com/OCA/OpenUpgrade/blob/8.0/addons/product/migrations/8.0.1.1/post-migration.py#L84

> This can be done without subquery

You're right, I'll change that. Thanks.